daemon/logger/loggerutils/rotatefilewriter.go

```diff
@@ -0,0 +1,127 @@
+package loggerutils
```

Comment:
I think this could go into `pkg`, maybe `ioutils` if we decide to make this satisfy `io.Writer`